docs/content/docs/start/quick-start.md

```diff
@@ -21,29 +21,18 @@ To get started you need the following installed.
 
 - [Node.js](https://nodejs.org/en/download) -- {{< include file="_includes/node-versions.md" >}}
 - Code editor -- we recommend [Visual Studio Code](https://code.visualstudio.com/).
-
-We also recommend that you install the following:
-
 - [Git](https://git-scm.com/downloads)
 
 ## Getting started
 
 Open a new command window and navigate to the folder you where you want to install the project, and then clone the
-[FluidHelloWorld repo](https://github.com/microsoft/FluidHelloWorld) with the following commands. The cloning process
+[FluidHelloWorld repo](https://github.com/microsoft/FluidHelloWorld) with the following command. The cloning process
 will create a subfolder named FluidHelloWorld with the project files in it.
 
 ```bash
 git clone https://github.com/microsoft/FluidHelloWorld.git
 ```
 
-{{< callout note >}}
-
-If you don't have Git installed you can [click here](https://github.com/microsoft/FluidHelloWorld/archive/main.zip) to
-download a zip of the FluidHelloWorld repo. Once the file downloads, extract the contents of the .zip file and run the
-following steps.
-
-{{< /callout >}}
-
```

Comment:
Curious on the reasoning behind removal? Here it likely makes sense, but do we have some general guidance on callouts/hints?